src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -265,10 +265,19 @@ def _forward(self, model_inputs):
             # it here.
             # Consume values so we can let extra information flow freely through
             # the pipeline (important for `partial` in microphone)
-            input_features = model_inputs.pop("input_features")
-            attention_mask = model_inputs.pop("attention_mask")
+            if "input_features" in model_inputs:
+                inputs = model_inputs.pop("input_features")
+            elif "input_values" in model_inputs:
+                inputs = model_inputs.pop("input_values")
+            else:
+                raise ValueError(
+                    "Seq2Seq speech recognition model requires either a "
```

Comment:
```suggestion
                    "The speech recognition model requires either an "
```
We're probably talking not only about seq2seq here?

Reply:
It's under the `if self.type == "seq2seq":` condition, so yeah it's just the seq2seq speech recognition model